src/model/entities/test/part.spec.ts

```diff
@@ -1,10 +1,10 @@
 import { Part, PartInterface } from '../part'
 import { EntityMockFactory } from './entity-mock-factory'
 import { PartTimings } from '../../value-objects/part-timings'
-import { Piece } from '../piece'
+import { Piece, PieceInterface } from '../piece'
 
-describe('Part', () => {
-  describe('getTimings', () => {
+describe(`${Part.name}`, () => {
```

Comment:
We the string interpolation?
You can just pass in the attribute value, like so:
```typescript
describe(Part.name, () => {
```

Reply:
I just took what we did somewhere else :p 
Updated

Reply:
Great! 

Thats a remnant from some rules in Sofie-Core, where i originally found that string interpolation was needed for it to be happy. It was me who origianlly suggested that way, and since moving to the new repo, we didn't check if the string interpolation was still needed.

Updating the lines in the test files touched by my current In Review task.